products.d/slowroll.yaml

```diff
@@ -0,0 +1,143 @@
+id: Slowroll
+name: Slowroll
+# ------------------------------------------------------------------------------
+# WARNING: There are currently no translations at this time!
+# ------------------------------------------------------------------------------
+description: 'A slightly slower rolling release of openSUSE designed to update
+  less often than Tumbleweed but more often than Leap without forcing users to
+  choose between "stable" and newer packages'
+icon: Slowroll.svg
+# Again, currently no translations at this time.
+translations:
+  description:
+    ca:
+    cs:
+    de:
+    es:
+    fr:
+    id:
+    ja:
+    nb_NO:
+    pt_BR:
+    ru:
+    sv:
+    tr:
+    zh_Hans:
+software:
+  installation_repositories: # was only able to find x86_64 at the time of writing
+    - url: https://download.opensuse.org/slowroll/repo/oss/
+      archs: x86_64
+    - url: https://download.opensuse.org/slowroll/repo/non-oss/
+      archs: x86_64
+
+  mandatory_patterns:
+    - enhanced_base # seems to be a consistant pattern between all openSUSE versions
+  optional_patterns: null
+  user_patterns:
+    - basic-desktop
+    - gnome
+    - kde
+    - yast2_basis
+    - yast2_desktop
+    - yast2_server
+    - multimedia
+    - office
+  mandatory_packages:
+    - NetworkManager
+    - openSUSE-repos-Slowroll
+  optional_packages: null
+  base_product: openSUSE
+
+security:
+  lsm: apparmor
+  available_lsms:
+    apparmor:
+      patterns:
+        - apparmor
+    selinux:
+      patterns:
+        - selinux
+      policy: enforcing
+    none:
+      patterns: null
+
+storage:
+  space_policy: delete
+  volumes:
+    - "/"
+    - "swap"
+  volume_templates:
+    - mount_path: "/"
+      filesystem: btrfs
+      btrfs:
+        snapshots: true
+        read_only: false
+        default_subvolume: "0"
+        subvolumes:
+          - path: home
+          - path: opt
+          - path: root
+          - path: srv
+          - path: usr/local
+          # Unified var subvolume - https://lists.opensuse.org/opensuse-packaging/2017-11/msg00017.html
+          - path: var
+            copy_on_write: false
+          # Architecture specific subvolumes (didnt see anything other than x86_64)
```

Comment:
As you took inspiration from the Leap and Tumbleweed ones, I would keep the same list of architecture-specific subvolumes to keep them as similar as possible. So, in the future, we won't need to update the list if Slowroll runs on a different architecture.

Reply:
i tried to do that but i will double check

Reply:
sorry i see what you mean and so ill add them :)